src/BloomBrowserUI/bookEdit/js/bloomEditing.js

```diff
@@ -440,22 +440,23 @@ function MakeSourceTextDivForGroup(group) {
   else {
     $(divForBubble).remove();//no tabs, so hide the bubble
     return;
-  }
+    }
+
+    var showEvents = false;
+    var hideEvents = false;
+    var shouldShowAlways = true;
+
+  
+    if(checkMightCauseHorizontallyOverlappingBubbles(group)) {
```

Comment:
@JohnThomson can you give me line numbers for what you're concerned about? I'm not seeing it.